src/components/shared/Picture.tsx

```diff
@@ -1,50 +1,42 @@
-import React, { useState, Fragment } from 'react';
-import { Image, ActivityIndicator, StyleSheet, ImageSourcePropType } from 'react-native';
+import React, { useState, Fragment } from 'react'
+import { Image, ImageSourcePropType } from 'react-native'
 
-import config from 'config';
+import config from 'config'
+import { Spinner } from 'native-base'
 
-const { IMAGE_BASE_URL } = config;
+const { IMAGE_BASE_URL } = config
 
 type PictureProps = {
-  source: ImageSourcePropType;
-  uri?: string;
-  style?: any;
-};
+  source: ImageSourcePropType
+  uri?: string
+  style?: any
+}
 
 const Picture = ({ source, uri, style }: PictureProps) => {
-  const [loader, setLoader] = useState<boolean>(true);
+  const [loader, setLoader] = useState<boolean>(true)
 
   const getSourceForImage = (): ImageSourcePropType => {
     if (source) {
-      return source;
+      return source
     }
 
-    return uri?.match(/^https?/) ? { uri } : { uri: IMAGE_BASE_URL + uri };
-  };
+    return uri?.match(/^https?/) ? { uri } : { uri: IMAGE_BASE_URL + uri }
+  }
 
   const getStyle = () => {
-    return style ? style : { width: 100, height: 100 };
-  };
+    return style ? style : { width: 100, height: 100 }
+  }
 
   return (
     <Fragment>
-      <Image style={getStyle()} source={getSourceForImage()} onLoadEnd={() => setLoader(false)} />
-      {loader && <ActivityIndicator style={styles.loading} animating={loader} size="large" />}
+      <Image
+        style={getStyle()}
+        source={getSourceForImage()}
+        onLoadEnd={() => setLoader(false)}
+      />
+      {loader && <Spinner size="lg" accessibilityLabel="Loading" />}
```

Comment:
I think that even an accessibility label should be localized.
```suggestion
      {loader && <Spinner size="lg" accessibilityLabel={$t('common.loading')} />}
```